src/Predis.php

```diff
@@ -65,10 +65,11 @@ public function __construct(
      * @param string   $key
      * @param callable $compute
      * @param int|null $cacheTime
+     * @param bool     $shouldUpdate
      *
      * @return mixed
      */
-    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null)
+    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null, bool $shouldUpdate = false)
     {
         if (!$this->refresh) {
```

Comment:
Can it be a bit simplified like this?
```suggestion
    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null, bool $refresh = false)
    {
        if (!$this->refresh && !$refresh) {
```

Reply:
updated